src/Kernel/Character.class.st

```diff
@@ -686,7 +686,11 @@ Character >> isSpacingCombiningMark [
 Character >> isSpecial [
 	"Answer whether the receiver is one of the special characters"
 
-	^'+-/\*~<>=@,%|&?!' includes: self
+	('+-/\*~<>=@,%|&?!' includes: self) ifTrue: [ ^true ].
```

Comment:
Can't we add a method #specialCharacterCodes returning all of those and make RBScanner use it?

It would avoid duplicating the information.

Reply:
yes, I was thinking about that, too...